app/views/shared/_header.html.erb

```diff
@@ -24,6 +24,10 @@
                 <% end %>
               </li>
             <% end %>
+            <% if current_user&.admin? %>
+              <li class="divider"></li>
+              <li><%= link_to "Manage Projects", admin_projects_path %></li>
```

Comment:
I wouldn't want to remove the existing link right away, that would cause problems for people that are already used to its current position.

Having the full text "Manage Projects" should make the link easier to parse at a glance, as you get a clear description of the action, instead of having to infer what it does from the context of its parent menu.